sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -40,17 +41,13 @@ export const ContentPlacementContainerStyles = css`
         border-bottom: none;
     }
 
-    @media screen and (min-width: 700px) {
+    @media screen and (min-width: ${siteBreakpoints[1]}) {
         .framework_ContentPlacement {
             border-width: 0 var(--border-width) var(--border-width) 0;
         }
     }
 
-    @media screen and (min-width: 700px) and (max-width: 999px) {
-        :host {
-            --flow: row;
```

Comment:
The assignment was deleted here as well.

Reply:
Well that's just weird.  The only thing I actively changed in this file was the breakpoints.  I have no clue how those wound up getting removed.

Reply:
Reverted